### PR TITLE
Bump Node.js to v16

### PR DESCRIPTION
**Pull request by cmoesel**
Node version for FPL should match node version for SUSHI.

### CODE REVIEW

**Review by mint-thompson — DISMISSED** [review]
hooray for node 16!

File: README.md
Comment (jafeltra) [inline README.md:1]: I can comment on the file, but I still can't comment on a line outside of the diff 😕 

On line 165 in the "For Developers" > "Installation" section there is another reference to the Node version. I think the change you have at the top of the README can just be dropped in here as well.

**Review by jafeltra — CHANGES_REQUESTED** [review]
I noticed one other place that needs the updated Node version.

File: README.md
Comment (cmoesel) [reply README.md:1]: Thanks @jafeltra - addressed in 2795a04.

**Review by jafeltra — APPROVED** [review]

**Pull request merged**